src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -493,6 +493,7 @@ function MoneyRequestView({
                                     filename={receiptURIs?.filename}
                                     transaction={transaction}
                                     enablePreviewModal
+                                    readonly={readonly}
```

Comment:
This change is missing the case where the recipe cannot be edited, which has raised #49882